composer/algorithms/seq_length_warmup/README.md

```diff
@@ -60,8 +60,8 @@ from composer.algorithms import SeqLengthWarmup
 trainer = Trainer(model=model,
                   train_dataloader=train_dataloader,
                   eval_dataloader=eval_dataloader,
-                  max_duration='250ep',
-                  algorithms=[SeqLengthWarmup()])
+                  max_duration='25ep',
+                  algorithms=[SeqLengthWarmup(max_seq_length=64)])
```

Comment:
Can you confirm this with @alextrott16 ? iirc there were some reasons why it was a large length, but not 100% sure

Reply:
There are some constraints around the gap between min and max, and the duration, so that the warmup schedule works out, but will wait for Alex if the example needed to be a really long sequence length for some reason.